src/btree/bt_vrfy.c

```diff
@@ -217,7 +217,7 @@ __wt_verify(WT_SESSION_IMPL *session, const char *cfg[])
 			WT_WITH_PAGE_INDEX(session,
 			    ret = __verify_tree(session, &btree->root, vs));
 
-			WT_TRET(__wt_cache_op(session, NULL, WT_SYNC_DISCARD));
+			WT_TRET(__wt_cache_op(session, NULL, WT_SYNC_CLOSE));
```

Comment:
@keithbostic, I've reverted this one for now so it's doing a discard again.  Let's revisit: I don't see how keeping the tree in cache helps verify subsequent checkpoints because each one will be a separate tree in memory even if they share pages on disk...